aiohttp/connector.py

```diff
@@ -467,30 +481,57 @@ def family(self):
         return self._family
 
     @property
+    def cache_dns(self):
```

Comment:
Renamed `cache_dns` to `dns_cache`.
Do you suggest renaming ctor parameter also?